src/utils.c

```diff
@@ -557,3 +557,12 @@ void dogecoin_uitoa(int n, char s[])
     s[i] = '\0';
     dogecoin_str_reverse(s);
 }
+
+int dogecoin_network_enabled() {
+    #ifndef WITH_NET
+    #define RETURN false
+    #else
+    #define RETURN true
+    #endif
+    return RETURN;
```

Comment:
I'd suggest against having a define of `RETURN` here because it's going to be defined for the remaining scope of utils.c, not just local to the `dogecoin_network_enabled` function (the preprocessor stuff is very primitive and just executes top down without any real knowledge of the language itself)

Reply:
Fixed, thanks for the suggestion, I should've gone this route (updated version) anyway. :heart: